keyserver/README.md

```diff
@@ -0,0 +1,43 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install keyserver/coniksserver/main.go
```

Comment:
Having a bin called `main` is not idea.

We could rename main.go, or ask people to either,

```
cd keyserver/coniksserver/
go build
./coniksserver -h
```

or, 

```
go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
coniksserver -h
```